### ISSUE TITLE
How do I train a detector using orginal batchsize……

### ISSUE THREAD

**Issue by Xudangliatiger** [Megvii-BaseDetection/BorderDet#14]
When I use the --4 gpus, the training setting is automatically changed, i.e., iterations are changed from 90000 to 180000, as same as the smaller lr and batchsize.

If I have enough GPU memory, I can simply use the orginal training setting with larger batchsize like 16.

**Comment by FateScript** [comment]
> When I use the --4 gpus, the training setting is automatically changed, i.e., iterations are changed from 90000 to 180000, as same as the smaller lr and batchsize.
> 
> If I have enough GPU memory, I can simply use the orginal training setting with larger batchsize like 16.

Default config is used for 8-GPU, if only 4 GPUs is detected while training, the config will change automatically. You don't need to change the config.py manually, If 8 GPUs is detected, the config will not change.

**Comment by Xudangliatiger** [comment]
Thanks. Actually I want to use the 8 GPUs' config for 4 GPU, since that I had to design some experiments about normalization layers.
